src/musicbrainz/chromaprinter.cpp

```diff
@@ -12,6 +12,16 @@
 
 namespace
 {
+// Type declarations of *fprint and *encoded pointers need to account for Chromaprint API version
+// (void* -> uint32_t*) and (void* -> char*) changed in versions v1.4.0 or later -- alyptik 12/2016
+#if (CHROMAPRINT_VERSION_MINOR > 3) || (CHROMAPRINT_VERSION_MAJOR > 1)
+    typedef uint32_t* uint32_p;
+    typedef char* char_p;
```

Comment:
I apologize, but I am still unsure what you mean here; the link you posted has the nested lines at four spaces, which is  what i have here. Did you want me to indent the typedefs an additional 4 spaces? I guess the style guidelines here are unclear for me because the directives are inside the anonymous namespace. Thanks for your patience with this.